ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -30,12 +36,28 @@ export GOPATH=/tmp/goproject
 export GOCACHE=/tmp/gocache
 export GOROOT=/usr/local/go
 
-# check if the image includes the exectable binary. we prepare it for late usage in case we need to compile it in the step.
-if [ -f "/usr/bin/extended-platform-tests" ]; then
-    echo "extended-platform-tests already exist, or else please try to compile it"
+# compile extended-platform-tests if it does not exist.
+# if [ -f "/usr/bin/extended-platform-tests" ]; then
+if ! [ -f "/usr/bin/extended-platform-tests" ]; then
```

Comment:
Do we need to use the `-x` to test if this `/usr/bin/extended-platform-tests` is executable?

Reply:
here we want to check if it exist because we have two sources to build image. one is in tests-private repo which has this binary, and the other is in verification-test which has no this binary.
and more, I checked it and it is executable after compiling.

so, -f is OK.

Thanks